en/docs/includes/start-apk.md

```diff
@@ -37,7 +77,7 @@ this in K8s.
         helm install <chart-name> <repository-name>/apk-helm --version <version-of-APK> --set wso2.subscription.imagePullSecrets=<secret-name>
         ```
 	
-    ### Helm install for Mac(arm64)
+    #### Helm install for Mac(arm64)
```

Comment:
I am running Mac - Intel (AMD64), what is my command for this? I know I need to use the above command. But it is not clear for others.

Reply:
fixed and resolved